### ISSUE TITLE
Add a linter to make sure deprecated flag is also applied to children

### ISSUE THREAD

**Issue by saschanaz** [mdn/browser-compat-data#7348]
See https://github.com/mdn/browser-compat-data/pull/7311/files#r522266719

**Comment by vinyldarkscratch** [comment]
+1 for this.  Should we also extend this to `experimental`?

**Comment by saschanaz** [comment]
Good idea! I'll do that as a followup, as the current PR already modifies multiple files and I see some issues there.

**Comment by vinyldarkscratch** [comment]
I was going through the list of PRs, and I noticed that you have a linter written for preventing both experimental and deprecated together (#6813).  I'm thinking that it would make sense to combine all three linter functions (`deprecated` consistency, `experimental` consistency, and no `deprecated` and `experimental` together) into a single `test-status.js` file?

**Comment by saschanaz** [comment]
Yay good idea, just renamed both to `test-status.js`. I can rebase them once either one is merged.

**Comment by jculver050** [comment]
Close

**Comment by caugner** [comment]
Fixed via https://github.com/mdn/browser-compat-data/pull/24810.
